### ISSUE TITLE
[Enhance] Landing Pads/Redstone Charge

### ISSUE THREAD

**Issue by Validfire** [DarkholmeTenk/TardisModIssues#196]
This is my first time using GitHub so if I do something wrong, sorry.

What I'm thinking is allow Landing pads to give off a redstone charge when a TARDIS is ontop of one.
with the ability to either disable and enable this Via config files, or something on the console to toggle this. Just an idea

**Comment by FoxPotato** [comment]
For a moment I thought this was a duplicate but you already mentioned this on the forum of course.
I'll make the same suggestion I made there, maybe have an Interface which adds this. So it can be placed inside the TARDIS too, where it'll ofc give a redstone signal when the exterior has landed on the pad.

**Comment by Soyaro666** [comment]
or simply a redstone interface you can put a redstone signal at. land on a pad, pull the lever you placed beside your interface on the inside and the outside interface emits redstone.
maybe an alternate (or additional) support for project red cable bundles?
I like the possibilities this could provide

**Comment by FoxPotato** [comment]
If there was gonna be a red project/redstone interface I think it should be an interface on it's own. Because what if I don't have project red  installed (Which is usually the case) and I wanna check if I'm on a landing pad, but I'm also using the redstone part in another part of my TARDIS. Do like the idea of a project red interface tho.

**Comment by Soyaro666** [comment]
so we are requesting 3 new interfaces:
Landing Pad Interface: If a Tardis is on a Landing Pad and it's installed on the landing pad or in the tardis on an open roundel the Interface would emit a redstone signal
Redstone Interface: allows to route redstone signals through Roundels and Landing Pads
Bundle Interface: allows to route Project Red Cable Bundles through Roundels and Landing Pads

did I summary it correctly?

**Comment by FoxPotato** [comment]
That's basically it, maybe just have the redstone interface work with the bundle interface. The ME interface does the same I think, ae2 has coloured cables too right?

**Comment by Soyaro666** [comment]
I don't know anything about AEs current state but last time I used it (1 or 2 years ago) it had all 16 colors + uncolored/unisulated cables. but they didn't have any differences beside the color I think so it should be irrelevant. Like I said, I don't know anything about newer versions of that mod. Of course the Tardis can't handle the 16 colored redpower wires in a bundled cable or on their own as one because this would completely destroy their function...

**Comment by FoxPotato** [comment]
I don't think it would be too hard, not as hard as other ideas we've come up with :p

**Comment by Validfire** [comment]
I don't want to download a mod just to get a certain feature out of a different mod. I don't think anyone else does either. I don't think the Redstone signal would be too hard to implement, I'm no coder just it doesn't seem like that would be too hard. I think the hardest thing to implement with the original idea is the Toggle Switch, which could possibly be next to the "Landing Pad Lockdown" switch on the Temporal Engine. Just my opinion with whats been going on since I've been absent from this topic. That's all.

**Comment by FoxPotato** [comment]
If Dark added an interface there would be no need for a switch. It would give off a redstone signal if there is an exterior on the landing pad the interface is on. Or make a roundel that ahs the interface give off a redstone signal if the exterior has landed on a landing pad.

**Comment by Validfire** [comment]
So what you're saying is add a new Component?

**Comment by FoxPotato** [comment]
If you by component mean interface, yes.
Interfaces are the things that allow you to acces stuff like the internal tanks and capacitors via roundels. These also work on landing pads.

**Comment by Validfire** [comment]
Yes, Exactly what I mean. That would do great. I don't care about details as long as It doesn't result in me adding more mods to my already extensive Modpack

**Comment by FoxPotato** [comment]
I don't think Dark will ever make the TARDIS mod dependant on another mod, he does add compatibility like the computercraft interface. But those mods are not needed to play the mod properly :)

**Comment by Validfire** [comment]
Just I'd like the ability to Toggle it On/Off without having to dismantle that Roundel each time. Due to the power of redstone it can do great good and great bad, or sometimes both. I just don't want to see my base destroyed because I landed my TARDIS on a pad. However I'd also like the feature so I could automate things for example my lights.

**Comment by FoxPotato** [comment]
Maybe he could make it so that the landing pad lever on the engine, the one that stops you from connecting with the landing pad, also stops you giving off a redstone signal. I mean, it would make sense. Kinda.
If you wanted to make like a trap, there are still a lot of ways to do that.

**Comment by Validfire** [comment]
If you mean the "Landing Pad Lockdown" switch yes that would/could work. I wish we got some of Darks commentary on what he thinks about all this. Because I've got a feeling the core Idea of this will change to something completely different soon...

**Comment by FoxPotato** [comment]
He's probably asleep, he'll probably wake up soon. I hope.

**Comment by Validfire** [comment]
I'm in the USA, I dont know what Country/Timezone he's in but... Oh crap the topic is already changing

**Comment by FoxPotato** [comment]
He's in his own personal time zone I think. And just delete the last message, like I will do :p

**Comment by Soyaro666** [comment]
it already did. But that doesn't matter since your initial idea will always be around. but you're right, Darkholme needs to comment this idea. tbh I would be surprised if he refuses this since it shouldn't be a problem to implement it but could be verry useful.

**Comment by FoxPotato** [comment]
We didn't stray that far tbh, we refined the idea of the landing pad detector thing itself and came up with another interface which also involved redstone. There are other posts where we really were talking about other stuff :p

**Comment by Soyaro666** [comment]
that's true. I meant the two other ideas we built based on his. what ever, let's wait for Darkholme.
(meant... stange word... feels like it should be "meaned" but that looked wrong too... nah, doesn't matter, sorry)

**Comment by FoxPotato** [comment]
2? I can only think of the redstone interface and the landing pad interface. Oh well.

**Comment by Soyaro666** [comment]
well, I counted redstone support and bundled cable support as 2 ^^

**Comment by Validfire** [comment]
What I think happened is We put our Ideas together to create something better, I mean look at the Pineapple. It looks better than a Pine cone, and tastes better than an Apple. Just using that as an example...

**Comment by FoxPotato** [comment]
I know what you mean but that example made like 0 sense and it confused me for a moment.
Are you saying that pineapples were created by combining traffic cones and apples.

**Comment by Validfire** [comment]
Basically

**Comment by Validfire** [comment]
Ok, I just got another Idea similar to this. Add an Interface that makes that Roundel give off a Redstone charge. This could be useful for many things for example Railroads (If you're system of corridors is big enough) This could supply a steady power source without the ugliness of redstone torches. What do you guys think about this? (I'm still pro landing pad redstone source, just possibly this could be added too)

**Comment by FoxPotato** [comment]
In my opinion that sounds only useful for what you said, a railroad. Nothing else really useful.

**Comment by Soyaro666** [comment]
what's the difference between a redstone signal and a redstone charge?

**Comment by FoxPotato** [comment]
He means just a redstone torch ability, when applied to a roundel it acts like a redstone torch/block

**Comment by Soyaro666** [comment]
so it emit's a redstone signal with maximal strength...? why shouldn't we want to have both? It's a Tardis, why shouldn't it take a redstone signal to do this? as long as the signal can be used as usual redstone signal I'm fine with the solution
sorry if i'm not understandable. I have some problems with writing english right now. No idea why...

**Comment by FoxPotato** [comment]
It's just, it's adding an interfaces just to make stuff look nice. The other ones actually have a use. It just seems like an useless addition to me.

**Comment by Soyaro666** [comment]
if it has only optical features I think so too... @Validfire what is it useful for?

**Comment by FoxPotato** [comment]
" This could be useful for many things for example Railroads (If you're system of corridors is big enough)" But I cant think of any other reasons except you dont have to place a redstone torch or block.

**Comment by Validfire** [comment]
It was just an Idea, even so, It would come in handy for other mods. Scratch that out of the books.
